sys-fs/ncdu/ncdu-1.14.1.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A disk usage analyser with a ncurses interface"
+DESCRIPTION="Ncdu aims to be fast, simple and easy to use, \
+and should be able to run in any minimal POSIX-like environment \
+with ncurses installed."
+HOMEPAGE="https://dev.yorhel.nl/ncdu"
+COPYRIGHT="2007-2019 Yoran Heling"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://dev.yorhel.nl/download/ncdu-$portVersion.tar.gz"
+CHECKSUM_SHA256="be31e0e8c13a0189f2a186936f7e298c6390ebdc573bb4a1330bc1fcbf56e13e"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ncdu = $portVersion
+	cmd:ncdu = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libncurses$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf$secondaryArchSuffix
```

Comment:
Need to check on x86_64, on _x86 it doesn't need autoconf, but it needs "cmd:awk"